test_build_bday_book.py

```diff
@@ -0,0 +1,13 @@
+"""Verify if the function build_bday_book() correctly assembles a
+dictionary of usernames as the keys and corresponding birthdates as the
+values.
+"""
+import view_user_birthdate
+
+def test_build_bday_book():
+    """Assert correct contents of an assmbled bday_book."""
+    file_name = 'testfile4build_bday_book.txt'
+    assert view_user_birthdate.build_bday_book(file_name) == {'austinwiltshire':'4/20/1983',
```

Comment:
I want your dictionary to be from usernames to users, and your users to print smartly enough that they print the dates right, don't precalculate the dates.

Your design would not survive me reassigning someone's nationality, for instance, since i forgot what they originally were. If you keep the users around and format at the last second, you keep information around that makes future changes and use easier.